app/src/main/java/org/mozilla/fenix/onboarding/SyncCFRPresenter.kt

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.onboarding
+
+import android.content.Context
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+import mozilla.components.service.glean.private.NoExtras
+import org.mozilla.fenix.GleanMetrics.Onboarding
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.cfr.CFRPopup
+import org.mozilla.fenix.compose.cfr.CFRPopupProperties
+import org.mozilla.fenix.home.recentsyncedtabs.view.RecentSyncedTabViewHolder
+
+/**
+ * Delegate for handling sync onboarding CFR.
+ *
+ * @param context used for various Android interactions.
+ * @param recyclerView will serve as anchor for the sync CFR
+ */
+
+class SyncCFRPresenter(
+    private val context: Context,
+    private val recyclerView: RecyclerView,
+) {
+
+    private var syncCFR: CFRPopup? = null
+
+    /**
+     * Check if [view] is available to show sync CFR.
+     */
+    fun showSyncCFR() {
+        findSyncTabsView()?.let {
+            CFRPopup(
+                text = context.getString(R.string.sync_cfr_message),
+                anchor = it,
+                properties = CFRPopupProperties(
+                    indicatorDirection = CFRPopup.IndicatorDirection.DOWN
+                ),
```

Comment:
@Mugurell can you take a look at this and suggest any properties that you should apply with your recent changes? Thanks

Reply:
Sure, was waiting for https://github.com/mozilla-mobile/fenix/pull/26585 to be merged to ensure the apis will be there.
That patch comes to help support more anchoring and specifically help this patch match the Figma design - https://www.figma.com/file/7J6gh5D8DbGTExxskKdn3T/Mobile-Onboarding?node-id=1950%3A64966.

There are two important differences that I see versus Figma:
- The popup should be centered in it's anchor.
This behavior can be set by using the `popupAlignment` property with the `BODY_TO_ANCHOR_CENTER` value. (This is the default value for the CFR composable)
- The popup body should overlap the synced tab by 1px.
This behavior can be set by using the `popupVerticalOffset` property with a `-16.dp` value.
The offset is in relation to the anchor (to support up/down anchorings) so a negative value means the popup will get closer to the middle of the anchor. (And a positive value will mean it is getting further from the anchor)
The `-16.dp` value is the results of considering:
   - The `8.dp` vertical padding [set in `RecentSyncedTabViewHolder`](https://github.com/mozilla-mobile/fenix/blob/b29b5049aa9efd1a1bed5c69a1ad5ba1ae3c888d/app/src/main/java/org/mozilla/fenix/home/recentsyncedtabs/view/RecentSyncedTabViewHolder.kt#L34-L35)
   - The `7.dp` as t[he default height of the popup indicator ](https://github.com/mozilla-mobile/fenix/blob/19831e5c79c27813a647bce1c24898ddebe60544/app/src/main/java/org/mozilla/fenix/compose/cfr/CFRPopup.kt#L140)
   - `1 more dp` for the 1dp overlap with the synced tab.
   
This would mean that the configuration can look like 
```kotlin
CFRPopup(
    text = context.getString(R.string.sync_cfr_message),
    anchor = it,
    properties = CFRPopupProperties(
        indicatorDirection = CFRPopup.IndicatorDirection.DOWN,
        popupVerticalOffset = (-16).dp,
    )
)
```

to get

![image](https://user-images.githubusercontent.com/11428869/186644604-901ed507-f577-46b8-b224-a9f89026087d.png)